pallets/parachain-staking/src/lib.rs

```diff
@@ -1133,64 +1075,32 @@ pub mod pallet {
 
 		/// Temporarily leave the set of collator candidates without unbonding
 		#[pallet::call_index(11)]
-		#[pallet::weight(<T as Config>::WeightInfo::go_offline())]
+		#[pallet::weight(<T as Config>::WeightInfo::go_offline(1_000))]
```

Comment:
What's the long term solution for this limit? We could define yet-another runtime config for the max, then it could be safely enforced and used as an upper bound here.

Reply:
You could do the same but without a runtime constant (just define a complile-time one). I would suggest using a constant here instead of a magic number anyway.

The question is: do we want to enforce it when joining candidates? It could be used as a DoS vector...

Reply:
Yeah that would be my choice as well.